pdp/static/js/pdp_controls.js

```diff
@@ -131,7 +132,7 @@ function Colorbar(div_id, layer) {
 Colorbar.prototype = {
     constructor: Colorbar,
 
-    midpoint: function () {
+    calculate_midpoint: function () {
```

Comment:
I think this is appropriate to rename this function, as a way to clarify that it's actually doing something and it's a function, not a property.